settings.html

```diff
@@ -44,6 +44,16 @@ <h4>This will allow you to borrow books from archive.org</h4>
                     </label>
                 </div>
             </div>
+            <div id="show_option_news"  class="col-sm-3">
+                <div class="news" style="padding-top: 5px;">
+                    <h2 style="font-family: Gill Sans Light">Show News Recommendation</h2>
+                    <h4>This will allow you to show Web News -> TV News</h4>
```

Comment:
@anishsarangi please rebase your branch to master. Always do that before submitting a PR because while you were working, other changes that affect you may have happened.

In this particular case, please see how I changed the settings texts. I suggest that you change this title to
"Show Web News -> TV News Recommendations" and drop `h4` entirely as its redundant.